lib/rules/base.js

```diff
@@ -69,6 +75,32 @@ module.exports = class Base {
     return this._configResolver.editorConfig();
   }
 
+  get _moduleId() {
+    const moduleIdKey = `${this.constructor.name}:moduleId`;
+
+    if (!loggedModules.has(moduleIdKey)) {
+      this._console.log('The `_moduleId` property is deprecated. Please use `_filePath` instead.');
+
+      this.loggedModules.push(moduleIdKey);
+    }
+
+    return this[MODULE_ID];
+  }
```

Comment:
Same as above, we don't have a preexisting `_moduleId` prop that is available on rules so I guess we don't need these getters.

Reply:
filePath will be public after https://github.com/ember-template-lint/ember-template-lint/pull/1729/files

Reply:
Yep, removed.